project/home/models.py

```diff
@@ -60,9 +71,8 @@ def get_html_url(self):
         # Override default url settings, make font black for readability
         return f'<a href="{url}" style="color: #000;">{self.title}</a>'
 
-    def save(self, *args, **kwargs):
-        super().save(*args, **kwargs)
-        # Grant 'view_event' permission to the assigned user
-        assign_perm('view_event', self.user, self)
```

Comment:
Why was this removed?

Reply:
My bad! Thank you for catching the issues! 
Those issues happened after I tried to resolve merge conflicts
I accidentally removed it to fix my errors. Then, I just remembered that I had to reverse back before pushing!

Reply:
FYI, I reversed save() back and resolved the issue!